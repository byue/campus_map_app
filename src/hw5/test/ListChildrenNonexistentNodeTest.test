# Tests listing children of non-existent node in graph

# Create a graph
CreateGraph graph1

# List Children of non-existent node
ListChildren graph1 n1
