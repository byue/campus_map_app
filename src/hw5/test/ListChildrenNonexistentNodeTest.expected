# Tests listing children of non-existent node in graph

# Create a graph
created graph graph1

# List Children of non-existent node
nodes are not present in graph